modules/tinymce/src/core/main/ts/content/GetContentImpl.ts

```diff
@@ -16,15 +16,36 @@ const trimEmptyContents = (editor: Editor, html: string): string => {
   return html.replace(emptyRegExp, '');
 };
 
+const bogusAllRegExp = /<(\w+) [^>]*data-mce-bogus="(all|1)"[^>]*>/g;
+
+const getPlainTextContent = (editor: Editor, body: HTMLElement) => {
+  let content: Content;
+
+  const doc = editor.getDoc();
+  content = body.innerHTML.replace(bogusAllRegExp, Unicode.zeroWidth);
+
+  const parseBody = doc.createElement('body');
+  parseBody.style.position = 'fixed';
+  parseBody.style.left = '-9999999px';
+  parseBody.style.top = '0px';
+
+  const root = doc.documentElement;
+  root.appendChild(parseBody);
+  parseBody.innerHTML = content;
+
+  content = Zwsp.trim(Unicode.removeZwsp(parseBody.innerText));
```

Comment:
This is stripping the Zwsp characters twice, as `Zwsp.trim` and `Unicode.removeZwsp` are the same thing.